t/test_postgresql

```diff
@@ -14,13 +14,20 @@ fi
 if test -d $DIR; then
   rm -r $DIR
 fi
-initdb --auth-local=peer -N $DIR -U $USER
+initdb --auth-local=peer -N $DIR -U $(whoami)
```

Comment:
I prefer to keep the user variable which I am sure is ensured to exist. Better than calling an external command

Reply:
I ran into this trying to use the script in a container, so I disagree :-)

Reply:
[da interweb](https://stackoverflow.com/questions/19306771/how-can-i-get-the-current-users-username-in-bash) seems to agree that your approach can be slightly better with a tendency towards "id" :D

Reply:
Alright, let's go for that.